requirements.txt

```diff
@@ -8,11 +8,11 @@ ecdsa==0.13
 gmpy==1.17
 jsonrpc==1.2
 jsonrpclib==0.1.7
-https://github.com/lbryio/lbryum/tarball/master/#egg=lbryum
+https://github.com/lbryio/lbryum/tarball/update-protobuf/#egg=lbryum
```

Comment:
yes, I'll revert this commit after lbryum gets merged.  I needed to
reference that branch to test the build process in the meantime.

On Tue, Aug 16, 2016 at 11:12 AM, Jack Robison <notifications@github.com>
wrote:

> In requirements.txt
> <https://github.com/lbryio/lbry/pull/119#discussion_r74966970>:
>
> > @@ -8,11 +8,11 @@ ecdsa==0.13
> >  gmpy==1.17
> >  jsonrpc==1.2
> >  jsonrpclib==0.1.7
> > -https://github.com/lbryio/lbryum/tarball/master/#egg=lbryum
> > +https://github.com/lbryio/lbryum/tarball/update-protobuf/#egg=lbryum
>
> should this be master? or should that be a separate commit after lbryum
> gets merged?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lbryio/lbry/pull/119/files/cc64da3d6249942ad52420b282f52e0f31ed9282#r74966970>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACRRX0R7slkYXV-qz8cur_naTzBhMRcyks5qgeFygaJpZM4JhNpP>
> .
>